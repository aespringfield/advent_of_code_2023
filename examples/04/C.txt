Card 1: 41 48 83 86 17 | 83 86  6 31 17  9 48 53
Card 2: 13 32 20 16 61 | 61 30 68 82 17 32 24 19
Card 3:  1 21 53 59 44 | 69 82 63 72 16 21 14  1
Card 4: 41 92 73 84 69 | 59 84 76 51 58  5 54 83
Card 5: 87 83 26 28 32 | 88 30 70 12 93 22 82 36
Card 6: 31 18 13 56 72 | 74 77 10 23 35 67 36 11
Card 7:  1  2  3  4  5 |  6  7  8  9 5 11 12 13
Card 8: 14 15 16 17 18 | 14 20 21 22 23 24 25 26
Card 9: 27 28 29 30 31 | 32 33 34 35 36 37 38 39
Card 10: 40 41 42 43 44 | 45 46 47 48 49 50 51 52
