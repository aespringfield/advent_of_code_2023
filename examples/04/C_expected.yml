part_two: 37
